frontend/src/pages/Users/Users.js

```diff
@@ -132,6 +145,11 @@ function UserList(props) {
           header: "Created",
           cell: item => <DateView date={item.UserCreateDate} /> || "-",
           sortingField: "UserCreateDate"
+        },
+        {
+          id: "delete",
+          header: "Action",
+          cell: item => <Button className="action" onClick={() => {setState(['app', 'user', 'delete'], item); showDialog('deleteUser')}} >Delete</Button>|| "-",
```

Comment:
Might factor this out into a component of `UserActions`

Reply:
The component will look like this:
```
function UserActions({ user }) {
  return <SpaceBetween direction="horizontal" size="s">
    <Button className="action" onClick={() => {setState(['app', 'user', 'delete'], user); showDialog('deleteUser')}} >Delete</Button>
  </SpaceBetween>
}
```

Using the component will change the line above to this:
```
cell: item => <UserActions user={item} />,
```

Reply:
ok done